hacktoberfest/gradlew

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
I would be good to mention explicitly who the author is, so that the authors are also clear when the file is viewed out of context of the version control history.